.travis.yml

```diff
@@ -12,7 +12,7 @@ matrix:
       dist: precise
     - php: 5.6
       env: WP_VERSION=latest
-    - php: 7.0
+    - php: 7.1
```

Comment:
I think one of the `5.6` builds should probably be switched to `7.0` so that gets run. Also might be worth considering adding a 7.2 build (which is allowed to fail, since 7.2 is pre-release)